src/bug-filing/common/create-issue-details-builder.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { IssueDetailsBuilder } from './issue-details-builder';
+import { MarkupFactory } from './markup/markup-factory';
+
+export const createIssueDetailsBuilder = (markup: MarkupFactory): IssueDetailsBuilder => {
+    const getter = (environmentInfo: EnvironmentInfo, data: CreateIssueDetailsTextData): string => {
+        const result = data.ruleResult;
+
+        const text = [
+            `${markup.bold('Issue')}: ${markup.snippet(result.help)} (${markup.link(result.helpUrl, result.ruleId)})`,
```

Comment:
>`${markup.bold('Issue')}: ${markup.snippet(result.help)} (${markup.link(result.helpUrl, result.ruleId)})`, [](start = 12, length = 106)

Personally, I think I liked the old Azure DevOps style better (where there were line breaks between the section headers and the single-line contents); description fields are usually on the thin side in a bug viewer, so I prefer keeping the line lengths short. Maybe we should run it by Chelsea?

Reply:
I run it by Fernanda and she was ok with how it looks. Remember we don't have full control of how the content looks as it is ultimately rendered by Azure boards/Github